### PR TITLE
Clarifications to the admin api documentation

### PR DESCRIPTION
**Pull request by richvdh**
Clarify how to authenticate

path params are not the same thing as query params /cc @dklimpel

### CODE REVIEW

File: docs/admin_api/README.rst [hunk 1]
```diff
@@ -4,17 +4,21 @@ Admin APIs
...
+server admin. (Note that a server admin is distinct from a room admin.)
 
-Restarting may be required for the changes to register.
+A user can be marked as a server admin by updating the database directly, e.g.:
 
-Using an admin access_token
-###########################
+```sql
+UPDATE users SET admin = 1 WHERE name = '@foo:bar.com';
+```
```
Comment (clokep) [inline docs/admin_api/README.rst:17]: This is a reStructuredText file, I think the following will work:

```suggestion
.. code-block:: sql

    UPDATE users SET admin = 1 WHERE name = '@foo:bar.com';
```

File: docs/admin_api/README.rst [hunk 2]
```diff
@@ -4,17 +4,21 @@ Admin APIs
...
+
+A new server admin user can also be created using the
+`register_new_matrix_user` script.
```
Comment (clokep) [inline docs/admin_api/README.rst:20]:
```suggestion
``register_new_matrix_user`` script.
```

File: docs/admin_api/user_admin_api.rst [hunk 3]
```diff
@@ -125,10 +127,10 @@ This API returns information about a specific user account.
 
 The api is::
 
-    GET /_synapse/admin/v1/whois/<user_id> (deprecated)
     GET /_synapse/admin/v2/users/<user_id>
```
Comment (dklimpel) [inline docs/admin_api/user_admin_api.rst:130]: I am not sure if this is a mistake in the documentation.
`GET /_synapse/admin/v1/whois/<user_id>` lists informations about the connections and `GET /_synapse/admin/v2/users/<user_id>`  about the user. This are different APIs.

File: docs/admin_api/README.rst (hunk 1 above)
Comment (richvdh) [reply docs/admin_api/README.rst:17]: good spot. thanks.

**Comment by richvdh** [comment]
92100d3d fixes documentation for `GET /_synapse/admin/v2/users/<user_id>` which appears to have been completely bogus ever since it was written in #6495

**Pull request merged**